### ISSUE TITLE
Loading Quicklisp's setup.lisp Results in a Stack Overflow

### ISSUE THREAD

**Issue by Shinmera** [drmeister/clasp#146]
Fun times to be had for all!
```
Starting Boehm cclasp 0.3-test-10-367-g62a5971 ... loading image... it takes a few seconds
Top level.
> (load #p"~/quicklisp/setup.lisp")
*** Do something with proclaim ftype (FTYPE (FUNCTION (T T T &OPTIONAL T) T)
                                        ENSURE-EXPORTED)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       (T &KEY (:DIRECTORY BOOLEAN)
                                        (:WILDEN BOOLEAN)
                                        (:ENSURE-DIRECTORY BOOLEAN))
                                       T)
                                      RESOLVE-LOCATION)
*** Do something with proclaim ftype (FTYPE (FUNCTION (&OPTIONAL T) T)
                                      INITIALIZE-SOURCE-REGISTRY)
;;; Warning: Condition: Undefined function named QL-UTIL:SAFELY-READ
;;; Warning: Condition: Undefined function named QL-UTIL:SAFELY-READ
;;; Warning: Condition: Undefined function named QL-CONFIG::CONFIG-VALUE-FILE-PATHNAME
;;; Warning: Condition: Undefined function named QL-CONFIG::CONFIG-VALUE-FILE-PATHNAME
;;; Warning: Condition: Undefined function named QL-CONFIG::CONFIG-VALUE-FILE-PATHNAME
;;; Warning: Condition: Undefined function named QL-CONFIG::CONFIG-VALUE-FILE-PATHNAME
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::INIT-FILE-NAME
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::INIT-FILE-NAME
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::QUICKLISP-INIT-FILE-FORM
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::QUICKLISP-INIT-FILE-FORM
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::INIT-FILE-NAME
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::INIT-FILE-NAME-FOR
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::INIT-FILE-NAME
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::INIT-FILE-NAME-FOR
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::SUITABLE-LISP-INIT-FILE
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::WRITE-INIT-FORMS
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::WRITE-INIT-FORMS
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::SUITABLE-LISP-INIT-FILE
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::WRITE-INIT-FORMS
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::WRITE-INIT-FORMS
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::DIRECTORYP
;;; Warning: Condition: Undefined function named CORE:RMDIR
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::DIRECTORYP
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL:DIRECTORY-ENTRIES
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::DIRECTORYP
;;; Warning: Condition: Undefined function named QL-IMPL-UTIL::DIRECTORYP
;;; Warning: Condition: Undefined function named QL-NETWORK:OPEN-CONNECTION
;;; Warning: Condition: Undefined function named QL-NETWORK:CLOSE-CONNECTION
;;; Warning: Condition: Undefined function named QL-PROGRESS::ELAPSED-TIME
;;; Warning: Condition: Undefined function named QL-PROGRESS::UNITS-PER-SECOND
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-OCTET-VECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-OCTET-VECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-DOWNCASE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-DOWNCASE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-DOWNCASE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-DOWNCASE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-VECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-VECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-OCTET-VECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-OCTET-VECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-PROCESSOR
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-PROCESSOR
;;; Warning: Condition: Undefined function named QL-HTTP::MATCH
;;; Warning: Condition: Undefined function named QL-HTTP::MATCH
;;; Warning: Condition: Undefined function named QL-HTTP::MULTI-CMATCH
;;; Warning: Condition: Undefined function named QL-HTTP::MATCH
;;; Warning: Condition: Undefined function named QL-HTTP::MULTI-CMATCH
;;; Warning: Condition: Undefined function named QL-HTTP::MATCH
;;; Warning: Condition: Undefined function named QL-HTTP::PROCESS-ALL
;;; Warning: Condition: Undefined function named QL-HTTP::PROCESS-ALL
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-PROCESSOR
;;; Warning: Condition: Undefined function named QL-HTTP::PROCESS-ALL
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-PROCESSOR
;;; Warning: Condition: Undefined function named QL-HTTP::PROCESS-ALL
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-PROCESSOR
;;; Warning: Condition: Undefined function named QL-HTTP::CMATCH
;;; Warning: Condition: Undefined function named QL-HTTP::PROCESS-ALL
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-PROCESSOR
;;; Warning: Condition: Undefined function named QL-HTTP::CMATCH
;;; Warning: Condition: Undefined function named QL-HTTP::PROCESS-ALL
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-UNTIL-MATCHING
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-UNTIL-MATCHING
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-OCTET
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-OCTET
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-OCTET
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-OCTET
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-STRING
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-STRING
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-OCTET
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-OCTET
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-OCTET
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-OCTET
;;; Warning: Condition: Undefined function named QL-HTTP::FULL-PROXY-PATH
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-STRINGS
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-NEWLINE
;;; Warning: Condition: Undefined function named QL-HTTP::USER-AGENT-STRING
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-NEWLINE
;;; Warning: Condition: Undefined function named QL-HTTP::SINK-BUFFER
;;; Warning: Condition: Undefined function named QL-HTTP::FULL-PROXY-PATH
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-STRINGS
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-NEWLINE
;;; Warning: Condition: Undefined function named QL-HTTP::USER-AGENT-STRING
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-NEWLINE
;;; Warning: Condition: Undefined function named QL-HTTP::SINK-BUFFER
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-UNTIL-MATCHING
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-OCTETS
;;; Warning: Condition: Undefined function named QL-HTTP::SINK-BUFFER
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-UNTIL-MATCHING
;;; Warning: Condition: Undefined function named QL-HTTP::ADD-OCTETS
;;; Warning: Condition: Undefined function named QL-HTTP::SINK-BUFFER
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-VECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MATCHES-AT
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-VECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MATCHES-AT
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-SUBSEQ
;;; Warning: Condition: Undefined function named QL-HTTP::HEADER-VALUE-INDEXES
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-SUBSEQ
;;; Warning: Condition: Undefined function named QL-HTTP::HEADER-VALUE-INDEXES
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-SUBSEQ
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-SUBSEQ
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-HEADER-VALUE
;;; Warning: Condition: Undefined function named QL-HTTP::ALL-FIELD-NAMES
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-HEADER-VALUE
;;; Warning: Condition: Undefined function named QL-HTTP::ALL-FIELD-NAMES
;;; Warning: Condition: Undefined function named QL-HTTP::HEADERS-ALIST
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-HEADER-VALUE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-HEADER-VALUE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-HEADER-VALUE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-HEADER-VALUE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-HEADER-VALUE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-HEADER-VALUE
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-DOWNCASE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-DOWNCASE
;;; Warning: Condition: Undefined function named QL-HTTP::SKIP-WHITE-FORWARD
;;; Warning: Condition: Undefined function named QL-HTTP::SKIP-WHITE-BACKWARD
;;; Warning: Condition: Undefined function named QL-HTTP::SKIP-WHITE-FORWARD
;;; Warning: Condition: Undefined function named QL-HTTP::SKIP-WHITE-BACKWARD
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-HVECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-HVECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-HVECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-HVECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::NEXT-LINE-POS
;;; Warning: Condition: Undefined function named QL-HTTP::STATUS-CODE
;;; Warning: Condition: Undefined function named QL-HTTP::FORCE-DOWNCASE-FIELD-NAMES
;;; Warning: Condition: Undefined function named QL-HTTP::CONTRACT-FIELD-VALUE-INDEXES
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-HVECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-HVECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-HVECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-HVECTOR
;;; Warning: Condition: Undefined function named QL-HTTP::NEXT-LINE-POS
;;; Warning: Condition: Undefined function named QL-HTTP::STATUS-CODE
;;; Warning: Condition: Undefined function named QL-HTTP::FORCE-DOWNCASE-FIELD-NAMES
;;; Warning: Condition: Undefined function named QL-HTTP::CONTRACT-FIELD-VALUE-INDEXES
;;; Warning: Condition: Undefined function named QL-HTTP::PARSE-URLSTRING
;;; Warning: Condition: Undefined function named QL-HTTP::PARSE-URLSTRING
;;; Warning: Condition: Undefined function named QL-HTTP:URL
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-REQUEST-BUFFER
;;; Warning: Condition: Undefined function named QL-HTTP::URLSTRING
;;; Warning: Condition: Undefined function named QL-HTTP:URL
;;; Warning: Condition: Undefined function named QL-HTTP:URL
;;; Warning: Condition: Undefined function named QL-HTTP:URL
;;; Warning: Condition: Undefined function named QL-HTTP:URL
;;; Warning: Condition: Undefined function named QL-HTTP:URL
;;; Warning: Condition: Undefined function named QL-HTTP::SINK-UNTIL-MATCHING
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE-MATCHER
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE-MATCHER
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE-MATCHER
;;; Warning: Condition: Undefined function named QL-HTTP::PROCESS-HEADER
;;; Warning: Condition: Undefined function named QL-HTTP::SINK-UNTIL-MATCHING
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE-MATCHER
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE-MATCHER
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE-MATCHER
;;; Warning: Condition: Undefined function named QL-HTTP::PROCESS-HEADER
;;; Warning: Condition: Undefined function named QL-HTTP::SINK-UNTIL-MATCHING
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE-MATCHER
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-SUBSEQ
;;; Warning: Condition: Undefined function named QL-HTTP::SINK-UNTIL-MATCHING
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE-MATCHER
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-SUBSEQ
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-STREAM-WRITER
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE-MATCHER
;;; Warning: Condition: Undefined function named QL-HTTP::READ-CHUNK-HEADER
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-FOR-N-OCTETS
;;; Warning: Condition: Undefined function named QL-HTTP::SKIP-UNTIL-MATCHING
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-STREAM-WRITER
;;; Warning: Condition: Undefined function named QL-HTTP::ACODE-MATCHER
;;; Warning: Condition: Undefined function named QL-HTTP::READ-CHUNK-HEADER
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-FOR-N-OCTETS
;;; Warning: Condition: Undefined function named QL-HTTP::SKIP-UNTIL-MATCHING
;;; Warning: Condition: Undefined function named QL-HTTP::CONTENT-LENGTH
;;; Warning: Condition: Undefined function named QL-HTTP::CHUNKEDP
;;; Warning: Condition: Undefined function named QL-HTTP::SAVE-CHUNK-RESPONSE
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-FOR-N-OCTETS
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-STREAM-WRITER
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-UNTIL-END
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-STREAM-WRITER
;;; Warning: Condition: Undefined function named QL-HTTP::CONTENT-LENGTH
;;; Warning: Condition: Undefined function named QL-HTTP::CHUNKEDP
;;; Warning: Condition: Undefined function named QL-HTTP::SAVE-CHUNK-RESPONSE
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-FOR-N-OCTETS
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-STREAM-WRITER
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-UNTIL-END
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-STREAM-WRITER
;;; Warning: Condition: Undefined function named QL-HTTP:UNEXPECTED-HTTP-STATUS-URL
;;; Warning: Condition: Undefined function named QL-HTTP:UNEXPECTED-HTTP-STATUS-CODE
;;; Warning: Condition: Undefined function named QL-HTTP:TOO-MANY-REDIRECTS-COUNT
;;; Warning: Condition: Undefined function named QL-HTTP:TOO-MANY-REDIRECTS-URL
;;; Warning: Condition: Undefined function named QL-HTTP::MERGE-URLS
;;; Warning: Condition: Undefined function named QL-HTTP:URL
;;; Warning: Condition: Undefined function named QL-HTTP::READ-HTTP-HEADER
;;; Warning: Condition: Undefined function named QL-HTTP::READ-HTTP-HEADER
;;; Warning: Condition: Undefined function named QL-HTTP::CONTENT-LENGTH
;;; Warning: Condition: Undefined function named QL-HTTP::SAVE-RESPONSE
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-WITH-PROGRESS-BAR
;;; Warning: Condition: Undefined function named QL-HTTP::CONTENT-LENGTH
;;; Warning: Condition: Undefined function named QL-HTTP::SAVE-RESPONSE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-HEADER-VALUE
;;; Warning: Condition: Undefined function named QL-HTTP::MERGE-URLS
;;; Warning: Condition: Undefined function named QL-HTTP::MERGE-URLS
;;; Warning: Condition: Undefined function named QL-HTTP:URL
;;; Warning: Condition: Undefined function named QL-HTTP::READ-HTTP-HEADER
;;; Warning: Condition: Undefined function named QL-HTTP::READ-HTTP-HEADER
;;; Warning: Condition: Undefined function named QL-HTTP::CONTENT-LENGTH
;;; Warning: Condition: Undefined function named QL-HTTP::SAVE-RESPONSE
;;; Warning: Condition: Undefined function named QL-HTTP::CALL-WITH-PROGRESS-BAR
;;; Warning: Condition: Undefined function named QL-HTTP::CONTENT-LENGTH
;;; Warning: Condition: Undefined function named QL-HTTP::SAVE-RESPONSE
;;; Warning: Condition: Undefined function named QL-HTTP::ASCII-HEADER-VALUE
;;; Warning: Condition: Undefined function named QL-HTTP::MERGE-URLS
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       ((UNSIGNED-BYTE 32)
                                        (SIMPLE-ARRAY (UNSIGNED-BYTE 8) (*))
                                        FIXNUM)
                                       (UNSIGNED-BYTE 32))
                                      UPDATE-ADLER32-CHECKSUM)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION NIL
                                       (SIMPLE-ARRAY (UNSIGNED-BYTE 32) (256)))
                                      GENERATE-CRC32-TABLE)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       ((UNSIGNED-BYTE 32)
                                        (SIMPLE-ARRAY (UNSIGNED-BYTE 8) (*))
                                        FIXNUM)
                                       (UNSIGNED-BYTE 32))
                                      UPDATE-CRC32-CHECKSUM)
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::UPDATE-ADLER32-CHECKSUM
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::UPDATE-CRC32-CHECKSUM
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::UPDATE-ADLER32-CHECKSUM
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::UPDATE-CRC32-CHECKSUM
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::UPDATE-ADLER32-CHECKSUM
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::UPDATE-CRC32-CHECKSUM
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::UPDATE-ADLER32-CHECKSUM
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::UPDATE-CRC32-CHECKSUM
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::SLIDING-WINDOW-STREAM-WRITE-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::SLIDING-WINDOW-STREAM-WRITE-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::SLIDING-WINDOW-STREAM-WRITE-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::SLIDING-WINDOW-STREAM-WRITE-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-GET-BYTE
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-READ-BITS
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-READ-BITS
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-READ-BITS
;;; Warning: Condition: Undefined function named QL-GUNZIPPER::BIT-STREAM-READ-BITS
../../src/core/lisp.cc:1558 Stack is getting full currently at 7350960 bytes - warning at 7340032 bytes

"#8274/b@0x2aebc64903c9         hashTable.cc   966/0   (COMMON-LISP:GETHASH CLOS::SLOTS #<CORE:HASH-TABLE-EQL :count 22 :total-alist-entries 22 @0x2aebcc2fe388>  )
" 
"The following restarts are available:" 
"ABORT      a    Abort to REPL" 
Frame-8274-Dbg[1]>
```

**Comment by Shinmera** [comment]
Still there just the same on 20745853842bb99f56672dd406c4e6a54fa78748

**Comment by Shinmera** [comment]
New failure on 7fde5b78e98bf1af8e74f3884815a81db9eeec7e
```
> (load #p"~/quicklisp/setup.lisp")
*** Do something with proclaim ftype (FTYPE (FUNCTION (T T T &OPTIONAL T) T)
                                        ENSURE-EXPORTED)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       (T &KEY (:DIRECTORY BOOLEAN)
                                        (:WILDEN BOOLEAN)
                                        (:ENSURE-DIRECTORY BOOLEAN))
                                       T)
                                      RESOLVE-LOCATION)
*** Do something with proclaim ftype (FTYPE (FUNCTION (&OPTIONAL T) T)
                                      INITIALIZE-SOURCE-REGISTRY)
;;; Warning: Condition: Undefined function named QL-HTTP::MAKE-OCTET-VECTOR
;;; Warning: Condition: Undefined function named QL-HTTP:UNEXPECTED-HTTP-STATUS-URL
;;; Warning: Condition: Undefined function named QL-HTTP:UNEXPECTED-HTTP-STATUS-CODE
;;; Warning: Condition: Undefined function named QL-HTTP:TOO-MANY-REDIRECTS-COUNT
;;; Warning: Condition: Undefined function named QL-HTTP:TOO-MANY-REDIRECTS-URL
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       ((UNSIGNED-BYTE 32)
                                        (SIMPLE-ARRAY (UNSIGNED-BYTE 8) (*))
                                        FIXNUM)
                                       (UNSIGNED-BYTE 32))
                                      UPDATE-ADLER32-CHECKSUM)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION NIL
                                       (SIMPLE-ARRAY (UNSIGNED-BYTE 32) (256)))
                                      GENERATE-CRC32-TABLE)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       ((UNSIGNED-BYTE 32)
                                        (SIMPLE-ARRAY (UNSIGNED-BYTE 8) (*))
                                        FIXNUM)
                                       (UNSIGNED-BYTE 32))
                                      UPDATE-CRC32-CHECKSUM)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       ((UNSIGNED-BYTE 32)
                                        (SIMPLE-ARRAY (UNSIGNED-BYTE 8) (*))
                                        FIXNUM)
                                       (UNSIGNED-BYTE 32))
                                      UPDATE-ADLER32-CHECKSUM)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION NIL
                                       (SIMPLE-ARRAY (UNSIGNED-BYTE 32) (256)))
                                      GENERATE-CRC32-TABLE)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       ((UNSIGNED-BYTE 32)
                                        (SIMPLE-ARRAY (UNSIGNED-BYTE 8) (*))
                                        FIXNUM)
                                       (UNSIGNED-BYTE 32))
                                      UPDATE-CRC32-CHECKSUM)

Condition of type: SIMPLE-TYPE-ERROR
Slot CODE-SYMBOLS in structure DECODE-TREE only admits values of type (SIMPLE-ARRAY FIXNUM (*)).
Available restarts:
(use :r1 to invoke restart 1)

1. (TRY-RECOMPILING) Recompile deflate and try loading it again
2. (RETRY) Retry loading FASL for #<CL-SOURCE-FILE "quicklisp" "deflate" @0x2b10ca2783d1>.
3. (ACCEPT) Continue, treating loading FASL for #<CL-SOURCE-FILE "quicklisp" "deflate" @0x2b10ca2783d1> as having been successful.
4. (RETRY) Retry ASDF operation.
5. (CLEAR-CONFIGURATION-AND-RETRY) Retry ASDF operation after resetting the configuration.
6. (RESTART-TOPLEVEL) Go back to Top-Level REPL.

Broken at frame[337] TOP-LEVEL.
 File: #P"/home/linus/.cache/shirakumo/autobuild/clasp-testing/.autobuild/7fde5b78e98bf1af8e74f3884815a81db9eeec7e/build/clasp/Contents/Resources/lisp/kernel/lsp/top.lsp" (Position #21029)
>> 
```

**Comment by Shinmera** [comment]
Fixed on d13d2308ca2ac24ec49a3d0e58f21efa96f91b6c.
```
> (load #p"~/quicklisp/setup.lisp")
*** Do something with proclaim ftype (FTYPE (FUNCTION (T T T &OPTIONAL T) T)
                                        ENSURE-EXPORTED)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       (T &KEY (:DIRECTORY BOOLEAN)
                                        (:WILDEN BOOLEAN)
                                        (:ENSURE-DIRECTORY BOOLEAN))
                                       T)
                                      RESOLVE-LOCATION)
*** Do something with proclaim ftype (FTYPE (FUNCTION (&OPTIONAL T) T)
                                      INITIALIZE-SOURCE-REGISTRY)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       ((UNSIGNED-BYTE 32)
                                        (SIMPLE-ARRAY (UNSIGNED-BYTE 8) (*))
                                        FIXNUM)
                                       (UNSIGNED-BYTE 32))
                                      UPDATE-ADLER32-CHECKSUM)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION NIL
                                       (SIMPLE-ARRAY (UNSIGNED-BYTE 32) (256)))
                                      GENERATE-CRC32-TABLE)
*** Do something with proclaim ftype (FTYPE
                                      (FUNCTION
                                       ((UNSIGNED-BYTE 32)
                                        (SIMPLE-ARRAY (UNSIGNED-BYTE 8) (*))
                                        FIXNUM)
                                       (UNSIGNED-BYTE 32))
                                      UPDATE-CRC32-CHECKSUM)
T

```
